### DISCUSSION TITLE
How to use Regional Prompting Pipeline in sdxl?

### DISCUSSION THREAD

**Discussion by mafamily2496** [huggingface/diffusers#7089] [Q&A]
@patrickvonplaten

**Comment by vgabbo** [comment]
Is there already a solution? I can't find anything.

**Reply by asomoza** [reply]
not yet, in all this time, no one contributed the SDXL regional prompting community pipeline. I'll try to do a demo implementation with it when we have `modular diffusers` but it's not a priority.

We still welcome and help anyone from the community that wants to contribute it.

**Reply by vgabbo** [reply]
I am looking into Region Aware Generation, if I can eventually understand it and replicate I will try to contribute but I'm not that good at programming. Thanks for your answer, I was searching but could not understand if I was simply unlucky with searching skills ahaha

**Reply by asomoza** [reply]
I remember now that there's one that resembles regional prompting which is called [Stable Diffusion Mixture Tiling Pipeline SDXL](https://github.com/huggingface/diffusers/tree/main/examples/community#stable-diffusion-mixture-tiling-pipeline-sdxl) but is not the same, as you can only use tiles and not regions, but maybe it works in your use case.

Also I never understood why the regional prompting implementation was done so weird everywhere, the best UI solution for this is to just pass painted regions but people had to make it hard with columns, rows and the BREAK thing.

If you want to look for a working good implementation (to use or to learn), I believe InvokeAI has regional prompting with painted regions.
